### ISSUE TITLE
FR: set showConstraintMessages to true when stringifing validation errors

### ISSUE THREAD

**Issue by mjpieters** [momocow/webpack-userscript#92]
Minor nit: Please set the `showConstraintMessages` parameter to `ValidationError.toString()` to `true` when validating headers, here:

https://github.com/momocow/webpack-userscript/blob/5bca4c8fa68cd296d58e0c7ada967882c9641f6e/lib/features/validate-headers/impl.ts#L54-L56

If you use `err.toString(undefined, undefined, undefined, true)` there then the validator _message_ is shown rather than the name, which is a bit more informative. Compare

```
- property updateURL has failed the following constraints: isUrl 
```

with 

```
- property updateURL has failed the following constraints: updateURL must be a URL address
```

**Comment by mjpieters** [comment]
I'd have expected the issue to be closed automatically, because my PR referenced it with 'fixes'.

But, given that it was merged into the dev branch, perhaps you wanted to keep this open until it has been released?

**Comment by momocow** [comment]
Don't worry, I'll ensure it closed after it is released.

There will be a few more minor features along with this one in the next version.

**Comment by momocow** [comment]
Released in [v3.2.0](https://www.npmjs.com/package/webpack-userscript/v/3.2.0).
